### DISCUSSION TITLE
Known issues...

### DISCUSSION THREAD

**Discussion by JRosanowski** [dotnet/maui#24810] [General]
Is there a way the .Net Maui team or maybe the community can set up a list of known issues by library and platform rather than everyone having to search the Issues to find solutions once they hit a problem? That way if you were about to attempt something in Maui you could also see if there's any problems you might encounter. Or is there something out there already?

For example.
GraphicsView 
  iOS - ICanvas.DrawString will only work if you pass a rectangle that will fit the text. You also need to include extra space because ICanvas.GetStringSize doesn't return the correct string size on iOS.
  https://github.com/dotnet/maui/issues/8486
  
If I'd know this when I started trying to implement the functionality I would have saved a huge amount of time, especially given how slow iOS builds are for us.

**Comment by gautambjain** [comment]
There are 3.5k open issues here - https://github.com/dotnet/maui/issues. Plus many on stackoverflow etc. Do you think it is possible to list all of them in a single page?

Only way is to search your issue (and related ones) and read discussions and current status of each.

**Reply by JRosanowski** [reply]
Sorry I wasn't clearer but I'm not suggesting every issue. Just ones that are common problems that may be encountered and are a confirmed and known issue that has been backlogged. 
The problem is that when you start working on something you can only search for a problem once you know that the problem exists and you've spent time making sure it's not your code causing it. 

Another example is the Interpreter setting on iOS issue that causes crashes. After you've spent hours looking at your code you start hunting for something that might mention a crash and eventually you might find it. If it was listed as a known issue everyone would save a lot of time.
